packages/common/src/core/schemas/shared/subschemas.ts

```diff
@@ -53,6 +53,22 @@ export const ENTITY_LOCATION_SCHEMA = {
   default: { type: "none" },
 };
 
+/**
+ * defines the JSON schema for the map configuration settings
+ */
+export const ENTITY_MAP_SCHEMA = {
+  type: "object",
+  properties: {
+    itemId: {
```

Comment:
I understand that the full path to this property is `initiative.mapSettings.itemId`, but just a heads up that I was pretty confused when I first saw this. Out of context, I didn't know what kind of item we were talking about (and whether it was a self reference or not).

What if we made it something more explicit like `basemapItemId`?

Reply:
I see.  I'm not sure `basemapItemId` is accurate since it's any web map or web scene item, not necessarily a basemap.  We could do something like `portalItemId`?

Reply:
Or maybe `baseViewItemId`?